HHS/Head-Start-TTADP
# [TTAHUB-1029] Show all AR's on goals & objectives page, show all topics on approved reports table

## Description of change

Fixes the following bugs:
- Objectives on the RTR page seem to only show one AR under "Activity Reports" even though they have been included on multiple ARs.
- Topics for newly created objectives do not display in the approved reports table, only the "legacy" topics field

## How to test

Reuse an existing goal and objective (that already has a report) on a new activity report, then approve it. You will see topics in the approved report table for the objective you created. Look up that goal & objective in the RTR. You will see both reports.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1029

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
